deployment/cfn/application.py

```diff
@@ -474,8 +421,7 @@ def create_auto_scaling_resources(self, app_server_security_group,
                 HealthCheckGracePeriod=600,
                 HealthCheckType='ELB',
                 LaunchConfigurationName=Ref(blue_app_server_launch_config),
-                LoadBalancerNames=[Ref(app_server_lb),
-                                   Ref(backward_compat_app_server_lb)],
+                LoadBalancerNames=[Ref(app_server_lb)],
```

Comment:
Since we updated `create_load_balancers` to be singular on Line 360, let's be consistent.  `LoadBalancerNames` should be made singular here as well, unless we foresee expanding this out to include more loadbalancers in the future - in which case, we should revert Line 360.

Reply:
Actually `LoadBalancerNames` is [a troposphere parameter](https://github.com/cloudtools/troposphere/blob/6ea2be21e93627715902ea73ae004c5d3db9e379/troposphere/autoscaling.py#L303), not in our control.

I do not foresee more load balancers being made for this, so will keep `create_load_balancer` as is.